tests/test-sources/plugins/utils/neoclip.nix

```diff
@@ -0,0 +1,119 @@
+{
+  empty = {
+    plugins.neoclip.enable = true;
+  };
+
+  with-sqlite = {
+    plugins.neoclip = {
+      enable = true;
+
+      settings.enable_persistent_history = true;
+    };
+  };
```

Comment:
How is sqlite enabled? Is it a nixpkgs dependency?

Reply:
Ah, I see. Maybe add a comment along the lines of "we install sqlite-lua when this option is `true`"?